### PR TITLE
feed

### PR DESCRIPTION
**Pull request by YelenaMerzlyakova**
Justification choices made:

I named my table notes because I used a collective name for the entities in my table (cfr conventions sql) (I did change it to singular).

1. I use capital letters for my names because I was conditioned to write a capital letter at the beginning of a title and or a name, hence it's just a force of habit and has no real justification. reading about conventions I realized it was not done and I should use small letters instead. 

2. I chose to have an author, titles and text (or body of my note) because that's the things I want to gather and display. I have no time stamp just because... . Text or Note might have been better naming choices instead of body. I chose singular because every note will have one title and one author. My table is a collection of all these things hence singular. Also I don't want added complexion by making use of plurals, I want to keep it nice and clean.

3. I want my Id's to be unique and have a numeral value. I was told by another learner that the average is 6 because it will be enough to create unique numbers. I have found different examples which all had different number filled in making this an arbitrary choice. 

  Also : A Unique Constraint  uniquely identifies each record in a database table. The unique 
  constraint prevents two records from having identical values in a specific column. 

4. The varchar of my body is set to 255 ( I heard somewhere that it's the standard amount of characters used, might have to look that one up again). 

5. The varchar of my title is set to 230, I'm contemplating setting it to 64 (yes a random amount, it looks round and happy) because notes shouldn't have a long title. The whole point of a note is that is short and on point. 

6. The varchar of my author is set to 100 but I will change it to 255  because 255 is a good baseline definition unless you particularly wish to curtail excessive input.

7. Not Null point to the fact that I don't want any blank parts on my note because all elements are important to deliver a good message. Take one away and you it will be less comprehensible.

### CODE REVIEW

**Review by Ilger — APPROVED** [review]
Statments always end with `;`

File: tablesql.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+--Use create database statement
+
+--CREATE DATABASE IF NOT EXISTS Notes;          --==> a query to create a database with the name Notes
+
+
+--To store the data in a table you you create another query by using a CREATE TABLE statement.
+
+CREATE TABLE IF NOT EXISTS Notes (            --==> a query to create a table 
+    NotesID   Int(6)  NOT NULL UNIQUE,
```
Comment (nicplackle) [inline tablesql.sql]: 'Int' and 'varchar' should be in all capitals

**Review by ZukisaNante — COMMENTED** [review]
Task completed! You may proceed!

**Review by ParwanySoman — COMMENTED** [review]
Looks clean and understandable! Nicely done!

**Comment by ParwanySoman** [comment]
I read your explanation and it all makes sense. Nice update!!

**Comment by KennethVDS** [comment]
It all looks very well explained and you have a nice straightforward way of thinking.

**Pull request merged**